http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpSamDAO.scala

```diff
@@ -383,6 +383,31 @@ class HttpSamDAO[F[_]](httpClient: Client[F],
       )(onError)
     } yield resp
   }
+
+  override def isGroupMembersOrAdmin(groupName: GroupName,
+                                     workbenchEmail: WorkbenchEmail)(implicit ev: Ask[F, TraceId]): F[Boolean] =
+    for {
+      leoAuth <- getLeoAuthToken
+      members <- httpClient.expectOr[List[WorkbenchEmail]](
+        Request[F](
+          method = Method.GET,
+          uri = config.samUri.withPath(Uri.Path.unsafeFromString(s"/api/groups/v1/${groupName.asString}/member")),
+          headers = Headers(leoAuth)
+        )
+      )(onError)
+      res <- if (members.contains(workbenchEmail))
+        F.pure(true)
+      else
+        for {
+          admins <- httpClient.expectOr[List[WorkbenchEmail]](
```

Comment:
Would be nice if we didn't have to make a Sam call for each policy but I don't see a way in the Sam API

Reply:
I know...wish api supports that